### PR TITLE
the first change

### LINKED ISSUE #5: Please do two times code editing

### PR DESCRIPTION
**Pull request by GuoFan1996**
this is my first change

### CODE REVIEW

File: HelloWord.java [hunk 1]
```diff
@@ -9,5 +9,7 @@ public static void main(String[] args) {
 
         System.out.println("hello from Guo");
 
+        System.out.println("we created a new isuue ");
```
Comment (JunjiaWangUSF) [inline HelloWord.java]: issues -spelling wrong

Comment (GuoFan1996) [reply HelloWord.java]: ok I will make a new commit to fix it

**Review by JunjiaWangUSF — APPROVED** [review]
APPROVE

**Comment by GuoFan1996** [comment]
finish lab

**Pull request merged**